### PR TITLE
Bug 1800748: Re-enable monitoring tests in CI

### PR DESCRIPTION
**Pull request by dtaylor113**
Re-enabled Monitoring and Alertmanager integration tests

### **Monitoring**
* Also added Routing Label name validation tests

image

### **Alertmanager**
image

### CODE REVIEW

**Comment by spadgett** [comment]
/lgtm
looks like this is still failing alertmanager tests

**Comment by spadgett** [comment]
@dtaylor113 the tests are broken... we should troubleshoot before retesting

**Comment by dtaylor113** [comment]
> @dtaylor113 the tests are broken... we should troubleshoot before retesting

I know, but they are passing for me locally :-(  Trying to run ` test-prow-e2e.sh` locally, but it's failing on trying to log into my cluster as kubeadmin

**Comment by dtaylor113** [comment]
Changing to WIP as I investigate why monitoring CI integration tests fail but pass locally.  Debug artifacts/messages show 'displays the Alermanager Configuration Details page' failing, but do not give line number, so can't determine which step is failing on CI.  May push some debugging PRs.

**Comment by dtaylor113** [comment]
@spadgett, this is the flake I keep seeing with .clear() and .sendKeys(), seems like .clear() doesn't happen at times?  And of course this passes locally, but fails in CI :-)
image

https://github.com/openshift/console/blob/b48e3c969cd559ba1cf99b18182d99978ac76896/frontend/integration-tests/tests/alertmanager.scenario.ts#L76

**Comment by dtaylor113** [comment]
@spadgett, @benjaminapetersen, @jcaianirh,
Hi, I'm working on re-enabling these tests.  The problem is .clear() works when running integration tests locally, but fails during CI (it fails to clear old values).  Initially I couldn't get e2e/prow working locally so I spent a bunch of time trying different ways of doing a .clear() & .sendKeys() -all of them worked when running integration tests locally, but continued to fail during CI.
I finally got e2e/prow working locally, and now I can see locally that clear() doesn't work after executing these instructions: https://github.com/openshift/console#how-the-integration-tests-run-in-ci
I'm investigating why these steps would cause clear() to no longer work.   
Any insight greatly appreciated! -thanks

**Comment by dtaylor113** [comment]
Added a 'wait until clickable` before clear().  CI failed 2 more times with seemingly unrelated issues.  Passed the 3rd try.  Going to retest again to verify.

**Comment by dtaylor113** [comment]
passed CI for 2nd time, retesting once more to be sure

**Comment by dtaylor113** [comment]
@spadgett, @benjaminapetersen, passed CI three times, removing WIP label -thanks

File: frontend/integration-tests/tests/alertmanager.scenario.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ import * as horizontalnavView from '../views/horizontal-nav.view';
 import { execSync } from 'child_process';
 
 const replaceInput = async (fieldName: string, value: string) => {
+  await monitoringView.wait(until.elementToBeClickable(firstElementByTestID(fieldName)));
```
Comment (spadgett) [inline frontend/integration-tests/tests/alertmanager.scenario.ts]: We shouldn't have a `monitoringView.wait`. We should just call `browser.wait` and use the default timeout. There's nothing specific to the monitoring view in that helper.

Comment (dtaylor113) [reply frontend/integration-tests/tests/alertmanager.scenario.ts]: ok, fixed

**Comment by benjaminapetersen** [comment]
/hold

Flakes

**Comment by benjaminapetersen** [comment]
/hold cancel

seems unrelated

**Pull request merged**